src/gluonts/torch/model/estimator.py

```diff
@@ -23,6 +23,7 @@
 from gluonts.env import env
 from gluonts.itertools import Cached
 from gluonts.model.estimator import Estimator
+from gluonts.nursery.utils import matching_parameters
```

Comment:
Better not to have imports from the nursery, maybe you could have this util elsewhere in the package?

Reply:
Done.